mail/alpine/Portfile

```diff
@@ -3,23 +3,23 @@
 PortSystem          1.0
 
 name                alpine
-version             2.21
+version             2.22
 revision            1
```

Comment:
The `revision` line needs to be reset to 0 when upgrading the `version`.

Reply:
Yeah, I just noticed that.  I spent long enough reminding people about that with bitrig ports I should know better :)